pkgs/top-level/aliases.nix

```
@@ -1408,6 +1408,7 @@ mapAliases ({
   ### Z ###
 
   zabbix40 = throw "'zabbix40' has been removed as it has reached end of life"; # Added 2024-01-07
+  zed-editor = throw "The version of 'zed-editor' previously in nixos-24.05 is outdated and should not be used (and newer versions cannot be backported as they require rust>=1.80 which is not present on this channel). Please install it from a more recent nixpkgs channel.";
```

"should not be used" implies that it "could be" used, if someone wants it.

Explicitly mention that `zed-editor` has been removed from nixos-24.05.

```suggestion
  zed-editor = throw "'zed-editor' has been removed from nixos-24.05 since it was outdated (and newer versions cannot be backported as they require rust>=1.80 which is not present on this channel). Please install it from a more recent nixpkgs channel like nixos-unstable.";
```

---

You are right. Done.